src/Main.js

```diff
@@ -1,18 +1,44 @@
 import React from "react";
-
-
 import HornedBeast from "./HornedBeast.js";
-
+import Form from 'react-bootstrap/Form';
+import Data from './data.json';
 
 import "./Main.css";
+import { Container } from "react-bootstrap";
 
 class Main extends React.Component {
-  render() {
 
-    console.log(this.props)
+  constructor(props) {
+    super(props);
+    this.state = {
+      everyBeast: Data,
+      seeBeast: Data
+
+    }
+  }
+
+  handleSelect = (hornNum) => {
+
+    console.log(typeof hornNum);
+
+    hornNum = parseInt(hornNum);
 
+    let updateBeast = this.state.everyBeast.filter(beast => beast.horns === hornNum ? beast : false);
 
-    let beastMap = this.props.data.map((beast, index) => (
+    console.log(updateBeast);
+
+    this.setState({
+      seeBeast: updateBeast
+    })
+
+    console.log(typeof hornNum);
+
+  };
+
```

Comment:
- We can do some cleanup of whitespace for this function and ideally it might look something like this for easier readability👍 

```
handleSelect = (hornNum) => {
    hornNum = parseInt(hornNum);
    let updateBeast = this.state.everyBeast.filter(beast => beast.horns === hornNum ? beast : false);

    let beastMap = this.props.data.map((beast, index) => (
      this.setState({
        seeBeast: updateBeast
      });
  };